src/components/create-from-builder/create-from-builder.controller.ts

```diff
@@ -233,4 +224,192 @@ export class CreateFromBuilderController implements angular.IController {
     let name = this.ctrl.imageStream.resource.metadata.name + ":" + this.ctrl.istag.name;
     return this.DataService.get("imagestreamtags", name, { namespace: 'openshift' });
   }
+
+  private preselectService() {
```

Comment:
Agreed, however the two were different enough that I did not create a binding service. I can add an issue to create a service in common to handle this.